src/frontends/onnx/tests/onnx_import_com_microsoft.in.cpp

```diff
@@ -18,6 +18,7 @@
 #include "common_test_utils/test_case.hpp"
 #include "onnx_import/onnx.hpp"
 #include "onnx_utils.hpp"
+#include "ngraph/file_util.hpp"
```

Comment:
As removing old API it should be added?

Reply:
We have the same API in openvino::util library.

ONNX tests should be migrated to new API by ONNX team